### ISSUE TITLE
Color for window counter and highlighted items is the same

### ISSUE THREAD

**Issue by dwma** [NicoHood/arc-theme#96]
### Details

- Distribution (with version): Arch Linux
- Desktop environment (with version): Gnome 3.28
- Arc version/commit (don't put "latest"): 20180715-1
- GTK+ 3 version: 3.22.30-1
- Installation method (package or source): package

The color of windows counter indicators in the dash is the same as the color for highlighted items. I don't know if this is a design decision and it's intended but it makes it a bit harder to cycle through windows with the mouse.

![screenshot from 2018-07-16 23-12-50](https://user-images.githubusercontent.com/14095540/42792941-1521f058-894e-11e8-856a-88e77a3e4c3c.png)   ![screenshot from 2018-07-16 23-13-09](https://user-images.githubusercontent.com/14095540/42792942-18dd4918-894e-11e8-865a-2a880654aec4.png)

**Comment by NicoHood** [comment]
Where is the right screenshot taken from? Another theme? Is it a suggestion how it should look like?

I personally dont use the dash at all (using plank) so I cannot decide what would be the best here.

**Comment by NicoHood** [comment]
![image](https://user-images.githubusercontent.com/6888294/42828324-aeba6a84-89e8-11e8-86f4-3aca6d4cfe92.png)

This is how it looks to me. I am also using arch, but that is still different, right?

**Comment by dwma** [comment]
The screenshots I've taken are using the Arc shell theme. Your screenshot looks like the default Adwaita shell theme.

**Comment by NicoHood** [comment]
Yes, thats it! What was the advantage of the Arc shell theme again? Do we really need it?

**Comment by floodcode** [comment]
@dwma, sorry for offtop, but what icon theme are you using?

**Comment by dwma** [comment]
It's Papirus.

**Comment by drakkar1969** [comment]
Found a way to do this keeping the original arc colors:

Before:
![before](https://user-images.githubusercontent.com/18715287/52960522-4872b280-3399-11e9-9535-e07d16e472c0.png)

After:
![after](https://user-images.githubusercontent.com/18715287/52960542-53c5de00-3399-11e9-90a8-4de2bc1ccac1.png)
